std_msgs/Header header

float64[] positionLeft
float64[] positionRight
float64[] orientationLeft
float64[] orientationRight

float64[] linearVelocityLeft
float64[] linearVelocityRight
float64[] angularVelocityLeft
float64[] angularVelocityRight